### ISSUE TITLE
WebSocket Error on close()

### ISSUE THREAD

**Issue by jassems** [featherweight/featherweight#101]
- there's a persistent browser-side error on ws.close() which does not have any known negative functional implications, but is noteworthy
- console error message: `WebSocket connection to ws://<address>:<port> failed: One or more reserved bits are on: reserved1 = 1, reserved2 = 0, reserved3 = 0`
- possible cause: not receiving the proper response payload on close() ?

note: have not yet looked at kernel code to compare with the required spec on ws close.

**Comment by jassems** [comment]
Update
- only occurs with the rep socket (pub socket connection opens and closes without error)
- seeing this error periodically now - as in, every other time a websocket is opened and closed
- note: all activity is kicked off by the onopen handler, including the close operation eventually (i.e. should not be some explicit race condition)

**Comment by jassems** [comment]
This issue has been resolved for quite some time now; had to do with properly constructing / respecting the nanomsg message spec for REQ-REP sockets.  I contributed ftwws.js which still needs to find a happy home via a pull-request into the proper location... originally pull-requested under the src folder, however it's more recently been suggested that the `contrib` folder - used for extensions - would be best.  Will open an issue to myself to get this done.
